governance.md

```diff
@@ -153,49 +289,135 @@ are enumerated below.
 
 ## Other duty-specific github roles:
 - [**Github Org Owner**](https://github.com/orgs/kubernetes/people?utf8=%E2%9C%93&query=%20role%3Aowner):
-  can create repos, do ~any github action; the number of
+  can create repos, and do any github action; the number of
   owners shouldn't scale with the organization's growth, O(1), and optimally it
   should be less than 20 people who are very familiar with project workings and
   distributed across a few time zones and organizations The other repos will
   have distinct sets of people filling some of the above roles, also.
 
+## Procedural roles
+
+- **Champion**
```

Comment:
It exists in at least the incubator process today. I thought it was useful to describe all the functions/roles in one place.